cli/.golangci.yml

```diff
@@ -1,4 +1,3 @@
-# DONT EDIT. This file is synced from https://github.com/cloudquery/.github/.github
```

Comment:
How do we decide, in cases like these, whether to keep only one global `.golangci.yml` or one per module?

Reply:
Hmm, it's mostly based on the code requirements of the module. Plugins have some auto-generated code that might conflict with linting (we can also make the autogenerated code play nice with liniting).

Specifically in our case for CLI and plugins I think we can use a single file, and will follow up on it in this PR.

Reply:
I kept 1 file for the CLI and 1 file for the plugins as it got a bit confusing to have a single file for all (e.g. hard to understand why some directories are ignored and where they belong)